src/cc/arduino/plugins/wifi101/UpdaterJFrame.java

```diff
@@ -74,7 +81,7 @@ public void run() {
 	}
 
 	public UpdaterJFrame() {
-		setTitle("WiFi101 Firmware/Certificates Updater");
+		setTitle("WiFi Firmware/Certificates Updater");
```

Comment:
Maybe here we call it `WiFi101 / WiFiNINA Firmware/Certificates Updater`? Please check with Andrea.

Reply:
ok, done asked to Andrea and will change the name as you suggested